.github/workflows/test-suite.yml

```diff
@@ -14,11 +14,11 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.6", "3.7", "3.8"]
+        python-version: ["3.6", "3.7", "3.8", "3.9.0-rc.2 - 3.9"]
```

Comment:
I'll update that once 3.9.0 appears in https://github.com/actions/python-versions/blob/main/versions-manifest.json

Reply:
Great yup. For now let's get this merged anyways, and update that once possible.